main.html.in

```diff
@@ -252,6 +252,8 @@
         <button id="benchmark-startup-baseline">Build Benchmark Baseline</button>
         <label><input type="number" min="1" max="100" id="benchmark-num-rounds"> Rounds</label>
         <label><input type="number" id="benchmark-round-delay" min="0" max="20000"> Round Delay(ms)</label>
+        <label><input type="checkbox" id="benchmark-warm-startup">Benchmark warm startup</label>
+        <label><input type="number" id="benchmark-startfg-delay" min="0" max="20000"> Delay after starting BG MIDlet</label>
```

Comment:
> Exactly, and also because if you wait more time, results will be stabler.

I'd love to be able to detect bg midlet idleness automatically, but that's a harder problem to solve. This'll do for now.